templates/manifest.json

```diff
@@ -508,6 +508,31 @@
                 "src/main.js",
                 "src/vector_index_cache.js"
             ]
+        },
+        {
+            "id": "ts-start-bun",
+            "name": "getting_started_typescript_bun",
+            "label": "Start with TypeScript on Bun",
+            "category": "typescript",
+            "technologies": [
+                "bun",
+                "cheerio",
+                "axios"
+            ],
+            "description": "Scrape single page with provided URL with Axios and extract data from page's HTML with Cheerio.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/ts-start-bun.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 256,
```

Comment:
Suggestion: I'd consider giving it a GB for a better user experience, it will speedup the container start likely

Reply:
It's really fast as it is.